src/components/Calculator.js

```diff
@@ -1,57 +1,40 @@
 import React from 'react';
 import './style.css';
-class ResultIcone extends React.Component {
-    constructor(props){
-        super(props)
-    }
-    render(){
-        return <div className = "calculator-result"> <span> 0</span> </div>
-    }
-}
-class LigneOfButton extends React.Component {
+import { ResultIcone } from './resultIcone';
+import { LigneOfButton } from './ligneOfButton';
+import calculate from '../logic/calculate';
+class CalculatorDesign extends React.Component {
   constructor(props){
     super(props)
-    this.props = props;
+    this.state = {};
+    this.handleClick = this.handleClick.bind(this);
   }
-  render() {
-    const btnsText = this.props.textBtns;
-    let content;
-    const order = this.props.order;
-    if(btnsText.length === 4){
-      content = (
-        <div className = 'command-line' style = {{gridRow: `${order} / span 1`}}>
-          <div className ="simple-btn"><span>{btnsText[0]}</span></div>
-          <div className ="simple-btn"><span>{btnsText[1]}</span></div>
-          <div className ="simple-btn"><span>{btnsText[2]}</span></div>
-          <div className ="op-btn"><span>{btnsText[3]}</span></div>
-        </div>
-        );
-    }
-    if(btnsText.length === 3){
-      content = (
-        <div className = 'command-line' style = {{gridRow: `${order} / span 1`}}>
-          <div className ="merge-btn"><span>{btnsText[0]}</span></div>
-          <div className ="simple-btn"><span>{btnsText[1]}</span></div>
-          <div className ="op-btn"><span>{btnsText[2]}</span></div>
-        </div>
-        );
+  handleClick (e){
+    console.log('clicked...');
```

Comment:
- [ ] It would be better if you don't commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. Kindly fix this minor issue. 

😄 💯